src/main/java/org/broadinstitute/consent/http/db/ConsentDAO.java

```diff
@@ -96,74 +103,76 @@ void updateConsent(@Bind("consentId") String consentId,
 
-    @SqlUpdate("delete from consentassociations where consentId = :consentId")
+    @SqlUpdate("DELETE FROM consent_associations WHERE consent_id = :consentId")
     void deleteAllAssociationsForConsent(@Bind("consentId") String consentId);
 
-    @SqlQuery("select distinct(associationType) from consentassociations where consentId = :consentId")
+    @SqlQuery("SELECT DISTINCT (association_type) FROM consent_associations WHERE consent_id = :consentId")
     List<String> findAssociationTypesForConsent(@Bind("consentId") String consentId);
 
-    @SqlQuery("select * from consents where consentId not in (select c.consentId from consents c  inner join election e on e.referenceId = c.consentId )")
+    @SqlQuery("SELECT * FROM consents WHERE consent_id NOT IN (SELECT c.consent_id FROM consents c INNER JOIN election e ON e.referenceId = c.consent_id )")
     List<Consent> findUnreviewedConsents();
 
-    @SqlQuery("select requiresManualReview from consents where consentId = :consentId")
+    @SqlQuery("SELECT requires_manual_review FROM consents WHERE consent_id = :consentId")
     Boolean checkManualReview(@Bind("consentId") String consentId);
 
-    @SqlQuery("select c.consentId, c.dac_id, c.name, c.createDate, c.sortDate, c.groupName, c.updated, e.electionId, e.status, e.version, e.archived  " +
-            "from consents c inner join election e ON e.referenceId = c.consentId inner join ( "+
-            "select referenceId, MAX(createDate) maxDate from election e group by referenceId) electionView "+
-            "ON electionView.maxDate = e.createDate AND electionView.referenceId = e.referenceId AND e.status = :status")
+    @SqlQuery("SELECT c.consent_id, c.dac_id, c.name, c.create_date, c.sort_date, c.group_name, c.updated, e.electionId, e.status, e.version, e.archived" +
+            " FROM consents c INNER JOIN election e ON e.referenceId = c.consent_id" +
+            " INNER JOIN (SELECT referenceId, MAX(createDate) maxDate FROM election e GROUP BY referenceId) electionView"+
+            " ON electionView.maxDate = e.createDate AND electionView.referenceId = e.referenceId AND e.status = :status")
     @UseRowMapper(ConsentManageMapper.class)
     List<ConsentManage> findConsentManageByStatus(@Bind("status") String status);
 
-    @SqlQuery("select ca.consentId from consentassociations ca  where ca.dataSetId IN (<dataSetIdList>) ")
+    @SqlQuery("SELECT ca.consent_id FROM consent_associations ca WHERE ca.dataset_id IN (<dataSetIdList>)")
     List<String> getAssociationConsentIdsFromDatasetIds(@BindList("dataSetIdList") List<Integer> dataSetIdList);
 
-    @SqlUpdate("update consents set updated = :updated where consentId = :referenceId")
+    @SqlUpdate("UPDATE consents SET updated = :updated WHERE consent_id = :referenceId")
     void updateConsentUpdateStatus(@Bind("referenceId") String referenceId,
                                    @Bind("updated") Boolean updated);
```

Comment:
We don't have a DAO test for this, please add one.

Reply:
Added tests in ConsentDAO for deleteConsent, deleteAllAssociationsForConsent, and updateConsentUpdateStatus. There was already a testUpdateConsentUpdateStatus but it was not testing the actual method.